sparta/src/CommandLineSimulator.cpp

```diff
@@ -1130,7 +1139,7 @@ bool CommandLineSimulator::parse(int argc,
                 throw_report_deprecated = true;
                 ++i;
 
-            }else if (o.string_key == "pipeline-collection") {
+            }else if (o.string_key == "pipeline-collection" or o.string_key == "roi-pipeline-collection") {
```

Comment:
Both options share the if condition, so I can guard that only one of them is enabled.